module/VuFind/src/VuFind/Sitemap/Plugin/Index/CursorMarkIdFetcher.php

```diff
@@ -96,13 +95,15 @@ public function setupBackend(string $backend): void
      * @param string $backend      Search backend ID
      * @param string $cursorMark   String representing progress through set
      * @param int    $countPerPage Page size
+     * @param array  $filters      Filters to apply to the search
      *
      * @return array
      */
     public function getIdsFromBackend(
         string $backend,
         string $cursorMark,
-        int $countPerPage
+        int $countPerPage,
+        array $filters = []
```

Comment:
Does this need the default? Since the whole plugin was just added, I think changing the signature would be quite alright at this point.